test/CMakeLists.txt

```diff
@@ -78,10 +83,38 @@ macro(setup_test)
     endif()
 endmacro(setup_test)
 
+macro(setup_pp_test)
```

Comment:
Rather than duplicate this macro, maybe use `if(AMR_WIND_TEST_WITH_PYTHON)` where necessary in the original macro?

Reply:
realized I didn't need the modified version after all